docs/DeveloperGuide.md

```diff
@@ -275,173 +220,155 @@ _{Explain here how the data archiving feature will be implemented}_
+| `* * *`  | social worker | view the information of different people I have added | retrieve their information more quickly.                          |
+| `* * *`  | social worker | delete the information of people                      | remove the data of people that I no longer need to keep track of. |
+| `* * *`  | social worker | edit a person's information                           | keep their information up-to-date for future use.                 |
+| `* *`    | social worker | tag a person                                          | manage and organize people according to their needs.              |
+| `* *`    | social worker | filter through the list of people                     | navigate the persons list more efficiently.                       |
+| `* *`    | social worker | sort people                                           | view people in a more suitable order.                             |
+| `* *`    | social worker | compile important personal details                    | quickly disseminate information to these people.                  |
+| `* *`    | social worker | archive people that no longer need assistance         | keep my contact list focused on active cases.                     |
+| `* *`    | social worker | see the overall statistics for people                 | better determine the progress I have or need to make.             |
+| `* *`    | social worker | schedule appointments I have with these people        | view all my appointments in one place.                            |
+| `* *`    | social worker | determine which schemes a person is eligible for      | help this person apply for said schemes.                          |
+
 
 ### Use cases
 
 (For all use cases below, the **System** is the `SocialBook` and the **Actor** is the `user`, unless specified otherwise)
 
-**Use case: Add new family**
+**Use case: UC01 - Add new person**
 
 **MSS:**
 
-1. User enters the command to add a family with the specified details
-2. SocialBook adds the family and displays the newly added family
+1. User enters the command to add a person with their specified details.
+2. SocialBook adds the person and displays the newly added person with the rest of the unarchived people.
 
     Use case ends.
 
 **Extensions:**
 
-* 2a. SocialBook detects missing or invalid input.
+* 1a. SocialBook detects missing or invalid input.
 
-    * 2a1. SocialBook displays an error message and prompts the user to try again
+    * 1a1. SocialBook displays an error message that suggest what a correct input should look like.
 
-    * 2a2. User corrects the input and enters the command again
+    * 1a2. User corrects the input and enters the command again.
     
-        Steps 2a1-2a2 are repeated until the user enters the correct input
+        Steps 1a1-1a2 are repeated until the user enters a correct input.
 
         Use case ends.
```

Comment:
Should it resume to step 2 after they enter a correct input rather than ending?

Reply:
Right thnks!